apps/hip/utils.py

```diff
@@ -33,7 +33,7 @@ def get_most_recent_objects(object_count=10):
             updated_at=F("latest_revision_created_at"),
             type_of_object=Value("PAGE", output_field=CharField()),
             model_name=Value("page", output_field=CharField()),
-        )[:object_count]
+        )
```

Comment:
Ah, alas, this used to have `.order_by("-updated_at")` before the `[:object_count]`, but it must have been erased.
I was thinking the `.order_by` would be faster than sorting in Python, but given the low number of objects, maybe it doesn't matter

Reply:
Oops, hope I didn't accidentally delete the `order_by`. I think you're right about your approach being better (DB sorting rather than Python sorting). I'll go ahead and merge, but we can add that change to another PR later, if you'd like.

Reply:
Sounds good. I think I must have erased it at some point in time